### DISCUSSION TITLE
Google Auth without Copy-Pasted Token

### DISCUSSION THREAD

**Discussion by Firemanarg** [GodotNuts/GodotFirebase#209] [Q&A] [answered]
Is there a way to implement a login system that the user can login with a Google Account, only by clicking in one button, then being redirected to Google Auth Page, and back again (with validated token) directly to the application?

**Comment by WolfgangSenff** [comment, marked as answer]
This is now supported! https://github.com/GodotNuts/GodotFirebase/pull/205
